src/Symfony/Component/Process/Process.php

```diff
@@ -503,41 +508,49 @@ public function getIncrementalOutput()
     /**
      * Returns an iterator to the output of the process, with the output type as keys (Process::OUT/ERR).
      *
-     * @param bool $blocking    Whether to use a blocking read call.
-     * @param bool $clearOutput Whether to clear or keep output in memory.
+     * @param int $flags A bit field of Process::ITER_* flags.
      *
      * @throws LogicException in case the output has been disabled
      * @throws LogicException In case the process is not started
      *
      * @return \Generator
      */
-    public function getIterator($blocking = true, $clearOutput = true)
+    public function getIterator($flags = 0)
```

Comment:
One of the best advantage of this is to be able to pipe a process in another process.
We could use the same behavior as unix `|` and forward only stdout (use `self:: ITER_SKIP_ERR)` by default).

WDYT?